backend/controllers/questionController.js

```diff
@@ -40,7 +40,15 @@ const createQuestion = async (req, res) => {
 
     // Add new question to database
     const dateCreated = moment().tz('Asia/Singapore').format(); // Gets the current time in Singapore Time
-    const questionDataWithDateCreated = { ...questionData, dateCreated };
+    const questionDataWithDateCreated = {
+      title,
+      description,
+      difficulty,
+      topics,
+      examples, // Ensure this field is always included
+      constraints, // Ensure this field is always included
+      dateCreated
+    };
```

Comment:
would like to check how each question's answers will be stored ! should it be with `questionData` or stored separately

Reply:
Currently there's no plans to include the question answer as a field or to store it somewhere. This is because our collaborative space will not have a code execution engine (it's a nice to have).

The only place we would need answers for is our examples part, and in that case, the answer will be contained within the example field itself.

Reply:
understood ! thank u 👍